src/androidMain/kotlin/software/momento/kotlin/sdk/internal/InternalDataClient.android.kt

```diff
@@ -0,0 +1,91 @@
+package software.momento.kotlin.sdk.internal
+
+import com.google.protobuf.ByteString
+import grpc.cache_client.ECacheResult
+import grpc.cache_client._GetRequest
+import grpc.cache_client._GetResponse
+import grpc.cache_client._SetRequest
+import io.grpc.Metadata
+import software.momento.kotlin.sdk.auth.CredentialProvider
+import software.momento.kotlin.sdk.config.Configuration
+import software.momento.kotlin.sdk.exceptions.CacheServiceExceptionMapper
+import software.momento.kotlin.sdk.exceptions.InternalServerException
+import software.momento.kotlin.sdk.responses.cache.GetResponse
+import software.momento.kotlin.sdk.responses.cache.SetResponse
+import java.io.Closeable
+
+internal actual class InternalDataClient actual constructor(
```

Comment:
There's duplication here and the manager files with the JVM main module as well but @nand4011 and I decided that we'll flush these out once we have the basic SDK wired up